src/core/Button/style.ts

```diff
@@ -0,0 +1,27 @@
+import styled from "@emotion/styled";
+import { Button } from "@material-ui/core";
+import { getCorners, getSpacings, Props } from "../styles";
+
+export interface IsRounded extends Props {
+  isRounded?: boolean;
+}
+
+export const StyledButton = styled(Button, {
+  shouldForwardProp: (prop) => prop !== "isRounded",
```

Comment:
we can't forward custom prop `isRounded` to button element, otherwise React will complain about it!